dcs/task.py

```diff
@@ -844,6 +844,28 @@ def __init__(self, position: Point = Point(0, 0), groupids: List[int] = None,
         }
 
 
+class Disembarking(Task):
+    """Unload task for helicopters.
+
+    Helicopter group will land at the given coordinates and will unload the given groups.
+
+    :param position: :py:class:`dcs.mapping.Point` where to land and unload
+    :param groupids: list of groups to unload
+    """
+    Id = "Disembarking"
+
+    def __init__(self, position: Point = Point(0, 0), groupids: List[int] = None):
```

Comment:
Sorry for being picky, but it should be:
```suggestion
    def __init__(self, position: Point = Point(0, 0), groupids: Optional[List[int]] = None):
```